README.md

```diff
@@ -11,8 +11,49 @@ This code is based off of code from [egeriis/zipcelx](https://github.com/egeriis
 If you're looking for advanced functionality, [js-xlsx](https://github.com/SheetJS/js-xlsx) is a solid choice.
 
 ## Table of contents
-1. [Examples and How to Use](https://github.com/elliotstoner/xljsx-lite-examples/)
-2. [Contributing](https://github.com/elliotstoner/xljsx-lite/wiki/Contributing)
+1. [Installation](#installation)
+2. [Usage](#usage)
+3. [Examples](https://github.com/elliotstoner/xljsx-lite-examples/)
+4. [Contributing](https://github.com/elliotstoner/xljsx-lite/wiki/Contributing)
+5. [Issues](#issues)
+
+## Installation
+
+**Install via NPM**
+
+```
+npm i xljsx-lite --save
+```
+
+**Use standalone**
+
+Download and include `lib/standalone.js` in your project. This will expose a global `xljsxLite` function you can use to create and download spreadsheets.
+
+
+## Usage
+
+```js
+import { xljsxLite } from 'xljsx-lite';
+
+const config = {
+  filename: 'example-spreadsheet', // output filename, excluding extension
+  sheets: [
+    {
+      sheetname: 'sheet1',
+      data: [
+        ['Test', 'Data', 'Row 2'], // Each array in 'data' represents a row
```

Comment:
Can you change this to "Row 1" so we don't confuse anyone?

Reply:
My bad!